lms/static/sass/course/_auto-cert.scss

```diff
@@ -0,0 +1,32 @@
+.wrapper-auto-cert {
+    display: table;
+    width: 100%;
+    clear: both;
```

Comment:
I would prefer that you remove the float on the page h1 and let the DOM handle the clearing here (remove clear both).